### PR TITLE
IBX-9094: Added recipe for ibexa/product-catalog-date-time-attribute

### PR DESCRIPTION
**Pull request by tbialcz**
| :ticket: Issue | IBX-9094 |
|----------------|----------|

Description:
Added recipe for ibexa/product-catalog-date-time-attribute package

For QA:
ibexa/product-catalog-date-time-attribute package after installation should be configured correctly

Documentation:
This PR is part of the ibexa/product-catalog-date-time-attribute package documentation.

### CODE REVIEW

File: ibexa/product-catalog-date-time-attribute/4.6/config/ibexa_product-catalog-date-time-attribute.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Example configuration for a custom date and time attribute:
+
+#ibexa_product-catalog-date-time-attribute:
```
Comment (konradoboza) [inline ibexa/product-catalog-date-time-attribute/4.6/config/ibexa_product-catalog-date-time-attribute.yaml:3]: Also file name needs to be adjusted. Besides, I would provide some example configuration, similar to https://github.com/ibexa/recipes-dev/blob/master/ibexa/product-catalog-symbol-attribute/4.6/config/ibexa_product_catalog_symbol_attribute.yaml.
```suggestion
#ibexa_product_catalog_date_time_attribute:
```

File: ibexa/product-catalog-date-time-attribute/5.0/config/ibexa_product-catalog-date-time-attribute.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Example configuration for a custom date and time attribute:
+
+#ibexa_product-catalog-date-time-attribute:
```
Comment (konradoboza) [inline ibexa/product-catalog-date-time-attribute/5.0/config/ibexa_product-catalog-date-time-attribute.yaml:3]: Same remark as above.

**Comment by konradoboza** [comment]
@tbialcz what is your take on this part from my comments? 😉  I don't see it addressed nor commented on:
>Besides, I would provide some example configuration, similar to https://github.com/ibexa/recipes-dev/blob/master/ibexa/product-catalog-symbol-attribute/4.6/config/ibexa_product_catalog_symbol_attribute.yaml.

**Comment by tbialcz** [comment]
@konradoboza My mistake, we don't have a configuration for date and time attribute, nor do we provide such an option; it was redundant.

**Review by konradoboza — APPROVED** [review]

**Review by adamwojs — APPROVED** [review]

**Review by Steveb-p — APPROVED** [review]

**Pull request merged**